src/pages/reference-guides/ReferenceGuideMenu.tsx

```diff
@@ -14,15 +15,27 @@ import SideMenuItemList from '../../widgets/menu/SideMenuItemList';
 import bundle from './ReferenceGuideMenu.nls';
 
 interface ReferenceGuideMenuProperties extends I18nProperties {
+	name: string;
 	route: string;
 	repo: string;
 	branch?: string;
 	path: string;
+	standaloneMenu?: boolean;
 }
 
 export default class ReferenceGuideMenu extends I18nMixin(WidgetBase)<ReferenceGuideMenuProperties> {
+	protected onAttach() {
+		const item = this.registry.getInjector<Router>('router');
```

Comment:
Why do we need to hook on to this event and invalidate any time an outlet changes?

Reply:
Its the only way the parent menu item highlighting works after the initial load. For example, if you are on the `Basic Usage` page of the `Reference Guides` / `i18n` section, all 3 of those menu items should be highlighted. Since the `Reference Guides` and `i18n` items are not actually Links, they do not get invalidated on Outlet change. This was an unfortunate workaround for that issue.

Reply:
I'd like to have a think about how we can better do this as it really shouldn't need a lifecycle method (which then also causes us to have tech debt in the test with the `getRender` and `bind` to call it)

Reply:
I agree. I will make an issue to look into this as future work.